### PR TITLE
translation of 01-1 after "Computing gradients by backpropagation"

### PR DESCRIPTION
**Pull request by a-oz**

### CODE REVIEW

**Comment by Atcold** [comment]
I'm confused. Is this the final version of `01-1.md`?

**Comment by a-oz** [comment]
Hi Alf, this is the translation of "Computing gradients by backpropagation" and contents after that, so @lejinghu can merge the translated contents before.

**Comment by Atcold** [comment]
Can you please merge before sending a PR?
I mean, can you merge on your fork?

Thanks.

**Comment by Atcold** [comment]
Creating two PRs on the same file create issues. See below. Anyhow, fixing them now.
Screenshot 2020-03-02 13 55 47

File: docs/zh/01-1.md [hunk 1]
```diff
@@ -4,6 +4,38 @@ lang-ref: ch.01-1
 title: Motivation of Deep Learning, and Its History and Inspiration
 authors: Yunya Wang, SunJoo Park, Mark Estudillo, Justin Mae
 date: 27 Jan 2020
+translation-date: 24 Feb 2020
+translator: lejinghu, Mingyang Zhao
 ---
 
-This will be the translated version of [Motivation of Deep Learning, and Its History and Inspiration]({{ site.baseurl }}{% link chapters/01-1.md %}).
+### [Computing gradients by backpropagation](https://www.youtube.com/watch?v=0bMe_vCZo30&t=2336s)
```
Comment (Atcold) [inline docs/zh/01-1.md]: Please, translate the title.

File: docs/zh/01-1.md [hunk 2]
```diff
@@ -4,6 +4,38 @@ lang-ref: ch.01-1
 title: Motivation of Deep Learning, and Its History and Inspiration
```
Comment (Atcold) [inline docs/zh/01-1.md]: Plz, translate the title.

**Comment by Atcold** [comment]
Also, rebase on top of master. I cannot merge now.

**Comment by a-oz** [comment]
Sorry for the inconvenience and conflicts should be resolved now and I patched all the fixes.

File: docs/chapters/02-1.md [hunk 3]
```diff
@@ -6,10 +6,10 @@ date: 3 February 2020
 ---
 
 
-## Gradient Descent Optimization Algorithm
+## [Gradient Descent optimization algorithm](https://www.youtube.com/watch?v=d9vdh3b787Y&t=29s)
```
Comment (Atcold) [inline docs/chapters/02-1.md]: I'm confused. Why is this file modified?

File: docs/chapters/02-2.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ date: 3 February 2020
 ---
 
 
-## A Concrete Example of Backpropagation and Intro to Basic Neural Network Modules
+## [A concrete example of backpropagation and intro to basic neural network modules](https://www.youtube.com/watch?v=d9vdh3b787Y&t=3022s)
```
Comment (Atcold) [inline docs/chapters/02-2.md]: This one too.

File: docs/chapters/05-1.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
+---
```
Comment (Atcold) [inline docs/chapters/05-1.md]: This file has been added from scratch.

**Comment by Atcold** [comment]
It looks like you've edited 33 files.
Screenshot 2020-03-03 01 05 20

**Comment by a-oz** [comment]
Hi @Atcold , I only edited one file after rebasing. But at the top of the page there are others' commits in this PR, and I'm looking for how this happens now. 
image

**Comment by a-oz** [comment]
Ok, I solve it by switching the base and then switch back, this is kind of a bug of Github. It's normal now: 
image

**Comment by Atcold** [comment]
The file tree is now the following:
```
└── zh
    ├── README-ZH.md
    ├── about.md
    ├── index.md
    ├── week01
    │   ├── 01-1.md
    │   ├── 01-2.md
    │   ├── 01-3.md
    │   └── 01.md
    └── week02
        ├── 02-1.md
        ├── 02-2.md
        ├── 02-3.md
        └── 02.md
```
Can you rebase on top of `master`? If it's too complicated @ebetica can help out.

**Comment by a-oz** [comment]
> The file tree is now the following:
> 
> ```
> └── zh
>     ├── README-ZH.md
>     ├── about.md
>     ├── index.md
>     ├── week01
>     │   ├── 01-1.md
>     │   ├── 01-2.md
>     │   ├── 01-3.md
>     │   └── 01.md
>     └── week02
>         ├── 02-1.md
>         ├── 02-2.md
>         ├── 02-3.md
>         └── 02.md
> ```
> 
> Can you rebase on top of `master`? If it's too complicated @ebetica can help out.

Rebase and resolving both done!

**Comment by Atcold** [comment]
Sweet! Thank you!
